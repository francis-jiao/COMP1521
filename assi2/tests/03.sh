# create and delete a linked list
# beware that check only allows 1500 lines of output
./test2
