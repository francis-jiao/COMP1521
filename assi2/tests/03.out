L = 83
+00000 (A,   16) +00016 (F, 9984) 
L = 83->86
+00000 (A,   16) +00016 (A,   16) +00032 (F, 9968) 
L = 77->83->86
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (F, 9952) 
L = 15->77->83->86
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (F, 9936) 

L = 15->77->83->86->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (F, 9920) 
L = 15->35->77->83->86->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (F, 9904) 
L = 15->35->77->83->86->86->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (F, 9888) 
L = 15->35->77->83->86->86->92->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (A,   16) +00128 (F, 9872) 
L = 15->35->49->77->83->86->86->92->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (A,   16) +00128 (A,   16) +00144 (F, 9856) 

L = 15->21->35->49->77->83->86->86->92->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (A,   16) +00128 (A,   16) +00144 (A,   16) 
+00160 (F, 9840) 
L = 15->21->35->49->62->77->83->86->86->92->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (A,   16) +00128 (A,   16) +00144 (A,   16) 
+00160 (A,   16) +00176 (F, 9824) 
L = 15->21->27->35->49->62->77->83->86->86->92->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (A,   16) +00128 (A,   16) +00144 (A,   16) 
+00160 (A,   16) +00176 (A,   16) +00192 (F, 9808) 
L = 15->21->27->35->49->62->77->83->86->86->90->92->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (A,   16) +00128 (A,   16) +00144 (A,   16) 
+00160 (A,   16) +00176 (A,   16) +00192 (A,   16) +00208 (F, 9792) 
L = 15->21->27->35->49->59->62->77->83->86->86->90->92->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (A,   16) +00128 (A,   16) +00144 (A,   16) 
+00160 (A,   16) +00176 (A,   16) +00192 (A,   16) +00208 (A,   16) +00224 (F, 9776) 

L = 15->21->27->35->49->59->62->63->77->83->86->86->90->92->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (A,   16) +00128 (A,   16) +00144 (A,   16) 
+00160 (A,   16) +00176 (A,   16) +00192 (A,   16) +00208 (A,   16) +00224 (A,   16) 
+00240 (F, 9760) 
L = 15->21->26->27->35->49->59->62->63->77->83->86->86->90->92->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (A,   16) +00128 (A,   16) +00144 (A,   16) 
+00160 (A,   16) +00176 (A,   16) +00192 (A,   16) +00208 (A,   16) +00224 (A,   16) 
+00240 (A,   16) +00256 (F, 9744) 
L = 15->21->26->27->35->40->49->59->62->63->77->83->86->86->90->92->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (A,   16) +00128 (A,   16) +00144 (A,   16) 
+00160 (A,   16) +00176 (A,   16) +00192 (A,   16) +00208 (A,   16) +00224 (A,   16) 
+00240 (A,   16) +00256 (A,   16) +00272 (F, 9728) 
L = 15->21->26->26->27->35->40->49->59->62->63->77->83->86->86->90->92->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (A,   16) +00128 (A,   16) +00144 (A,   16) 
+00160 (A,   16) +00176 (A,   16) +00192 (A,   16) +00208 (A,   16) +00224 (A,   16) 
+00240 (A,   16) +00256 (A,   16) +00272 (A,   16) +00288 (F, 9712) 
L = 15->21->26->26->27->35->40->49->59->62->63->72->77->83->86->86->90->92->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (A,   16) +00128 (A,   16) +00144 (A,   16) 
+00160 (A,   16) +00176 (A,   16) +00192 (A,   16) +00208 (A,   16) +00224 (A,   16) 
+00240 (A,   16) +00256 (A,   16) +00272 (A,   16) +00288 (A,   16) +00304 (F, 9696) 

L = 15->21->26->26->27->35->36->40->49->59->62->63->72->77->83->86->86->90->92->93
+00000 (A,   16) +00016 (A,   16) +00032 (A,   16) +00048 (A,   16) +00064 (A,   16) 
+00080 (A,   16) +00096 (A,   16) +00112 (A,   16) +00128 (A,   16) +00144 (A,   16) 
+00160 (A,   16) +00176 (A,   16) +00192 (A,   16) +00208 (A,   16) +00224 (A,   16) 
+00240 (A,   16) +00256 (A,   16) +00272 (A,   16) +00288 (A,   16) +00304 (A,   16) 
+00320 (F, 9680) 
After freeList ...
+00000 (F,10000) 
